packages/@aws-cdk/aws-codedeploy/test/ecs/appspec.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Ah that should not have been committed, will remove.

Generating the appspec is typically generated for each deployment, because the `TaskDefinition` field changes for every deployment.  IIUC, the changes for the `Deployment` construct in https://github.com/aws/aws-cdk/pull/22455 generate the appspec.